### ISSUE TITLE
Error in opening PDF file

### ISSUE THREAD

**Issue by sridhard** [mozilla/pdf.js#9934]
Attach (recommended) or Link to PDF file here:
https://d29xdxvhssor07.cloudfront.net/testfolder/video/error.pdf

Configuration:
- Web browser and its version: Chrome, 67.0.3396.99
- Operating system and its version: Mac OS 10.13.5 (17F77)
- PDF.js version: we tried with hosted demo
- Is a browser extension: No

Steps to reproduce the problem:
1. Just open the file using PDF JS, https://mozilla.github.io/pdf.js/web/viewer.html?file=https://d29xdxvhssor07.cloudfront.net/testfolder/video/error.pdf

What is the expected behavior? (add screenshot)
PDF should display success

What went wrong? (add screenshot)
PDF giving error

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):
https://mozilla.github.io/pdf.js/web/viewer.html?file=https://d29xdxvhssor07.cloudfront.net/testfolder/video/error.pdf

**Comment by timvandermeij** [comment]
The console shows `inconsistent generation in XRef`.

**Comment by sridhard** [comment]
Yes. But all other PDF viewers are able to display it. So it is better if pdfjs consider this and recover the broken xref.

Many pdf editors/writers are generating these kind of docs. The one we share is generated with foxit which is one of the popular writer/editor

**Comment by timvandermeij** [comment]
Indeed, we always try to recover data if we can. Marking as corrupted PDF for now so we can look into have to address this.
